### PR TITLE
SHARE-419 Tabwidth in Userview should not change size 

### PR DESCRIPTION
**Pull request by maxfazeli-dev**
For every tab in the Userview the number of projects, followers and following is now placed above the string and not on the left of the string.

---
### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroweb-Symfony/blob/develop/.github/contributing.md) and [wiki pages](https://github.com/Catrobat/catroweb-Symfony/wiki/) of this repository.

- [x] Include the name and id of the Jira ticket in the PR’s title eg.: `SHARE-666 The devils ticket`
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no warnings and errors 
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Verify that all tests are passing (CI), if not please state the test cases in the [section](#Tests) below
- [x] Perform a self-review of the changes
- [x] Stick to the project’s git workflow (rebase and squash your commits)
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] Put your ticket into the `Code Review` section in [Jira](https://jira.catrob.at/)
- [x] Post a message in the *#catroweb* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer
- [x] Check that your pull request has been successfully deployed to https://web-test-1.catrob.at/

### Additional Description
`TODO: Add additional information that is not in your commit-message here`

### Tests - additional information
`TODO: add additional information about testruns here`

### CODE REVIEW

**Review by dmetzner — CHANGES_REQUESTED** [review]
Prinzipiell wurden alle Akzeptanzkriterien des Tickets erfüllt.
Aber folgendes:

Manche Länder haben längere Texte, dann klappen die Tabs komplett ein:
image

Mein Vorschlag: Alles in einer Reihe lassen. Es sollte einfach scrollable sein. Ähnlich wie bei unseren Notification categories? Aber ohne chips.

Bitte neue LTMs für die Tabs einführen mit unterschied zwischen Singular und plural

image

**Comment by Tamdav** [comment]
Ich stimme @dmetzner zu. Bitte die Tabs scrollable machen und den Text in einer Zeile lassen. Siehe scrollable tabs: [material.io/tabs](https://material.io/components/tabs)

File: tests/behat/context/BrowserContext.php [hunk 1]
```diff
@@ -483,6 +483,27 @@ public function iWaitForTheElementToContain($locator, $text): void
...
 
+  /**
+   * @Then The element with id :selector should contain :text
+   *
+   * @param mixed $id
+   * @param mixed $text
+   *
+   * @throws ResponseTextException
+   */
+  public function tabShouldContain($id, $text): void
```
Comment (dmetzner) [inline tests/behat/context/BrowserContext.php]: This already exists in the Mink Context. 

Example usage in our project:     
```
And the "#app-version" element should contain "TEST_VERSION"
```

vendor/behat/mink-extension/src/Behat/MinkExtension/Context/MinkContext.php
```
    /**
     * Checks, that element with specified CSS contains specified HTML
     * Example: Then the "body" element should contain "style=\"color:black;\""
     * Example: And the "body" element should contain "style=\"color:black;\""
     *
     * @Then /^the "(?P<element>[^"]*)" element should contain "(?P<value>(?:[^"]|\\")*)"$/
     */
    public function assertElementContains($element, $value)
    {
        $this->assertSession()->elementContains('css', $element, $this->fixStepArgument($value));
    }
```

File: templates/UserManagement/Profile/profile.html.twig [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 {% block head %}
   <link rel="stylesheet" href="{{ asset('css/'~themeAssets()~'/profile.css') }}" media="screen"/>
+  <script src="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.js"></script>
```
Comment (dmetzner) [inline templates/UserManagement/Profile/profile.html.twig]: We don't want external URLs in our template. Instead, we want to include all dependencies directly in the project. (npm)

**Review by dmetzner — CHANGES_REQUESTED** [review]

**Review by dmetzner — APPROVED** [review]

**Pull request merged**